netsim/devices/linux.yml

```diff
@@ -29,7 +29,6 @@ group_vars:
 clab:
   image: python:3.9-alpine
   mtu: 1500
-  kmods:
```

Comment:
Should we remove it from all devices then, and assume containers need the system default kernel modules loaded (unless they specifically override the modules)?

Reply:
There are good reasons things are the way they are. Some of them are documented in past discussions or issues. In this particular case, some containers do not need extra drivers (SR Linux, Arista cEOS), some Linux distros don't have all the necessary drivers by default, some containers fail if the drivers are not loaded before the containers are started, and it's impossible to load drivers in GitHub Codespaces.

Also, stop changing things you don't need to change (and sometimes don't understand the implications of) just because you don't like them.